### DISCUSSION TITLE
immich_machine_learning for Nvidia Jetson (arm64)

### DISCUSSION THREAD

**Discussion by bogdanr** [immich-app/immich#13178] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a Jetson Orin Nano where I would like to run the machine learning container with CUDA support but the docker image is unavailable for this platform.

Having a docker image that we can run on Nvidia Jetson devices would be very nice.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10647
